lib/php/rendition-factory/src/Transformer/Video/FFMpegTransformerModule.php

```diff
@@ -43,7 +42,7 @@ public function transform(InputFileInterface $inputFile, array $options, Transfo
         throw new \InvalidArgumentException(sprintf('Invalid format %s', $format));
     }
 
-    private function doVideo(string $format, string $extension, InputFileInterface $inputFile, array $options, TransformationContext $context): OutputFileInterface
+    private function doVideo(string $format, string $extension, InputFileInterface $inputFile, array $options, TransformationContextInterface $context): OutputFileInterface
```

Comment:
I think you have a rebase issue (see https://github.com/alchemy-fr/phrasea/blob/master/lib/php/rendition-factory/src/Transformer/Video/FFMpegTransformerModule.php)